### PR TITLE
Add headers to all files / Create starter files

### PR DESCRIPTION
**Pull request by hannahsalameh**
Added a header to the starter index.html file and created starter files for all of the other pages I want contained in my portfolio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo Hannah Salameh's portfolio and STEP projects.
```
Comment (RegExrTech) [inline README.md]: I think you removed the word "contains" by mistake

**Review by RegExrTech — CHANGES_REQUESTED** [review]
Please review and resolve comments, thanks! (PS this is new to me too so hopefully I didn't mess any part of this up)

**Comment by RegExrTech** [comment]
Hey Hannah! Did you remove the existing .git file and create a new repo? It looks like this is still forked from the original repo but I can't be sure.

File: README.md (hunk 1 above)
Comment (hannahsalameh) [reply README.md]: I'll go in and fix that. Also I'm not sure how to respond to your other comments, but I am pretty sure this is in a new repo. It says its unlinked from the google repo at least.

Comment (RegExrTech) [reply README.md]: okay that sounds good then! Thanks!

**Review by RegExrTech — APPROVED** [review]

**Pull request merged**